### PR TITLE
Task/UI functionality/adminauth

### PR DESCRIPTION
**Pull request by whyhay01**

### CODE REVIEW

File: app/src/main/java/com/example/tapshopping/data/model/DummyData.kt
Comment (delaroy) [inline app/src/main/java/com/example/tapshopping/data/model/DummyData.kt:1]: take the dummy out

File: app/src/main/java/com/example/tapshopping/domain/ShoppingRepositoryImpl.kt
Comment (delaroy) [inline app/src/main/java/com/example/tapshopping/domain/ShoppingRepositoryImpl.kt:1]: incorporate the safe api call

File: app/src/main/java/com/example/tapshopping/ui/viewModel/AdminViewModel.kt
Comment (delaroy) [inline app/src/main/java/com/example/tapshopping/ui/viewModel/AdminViewModel.kt:1]: use the viewmodel scope

File: app/src/main/res/layout/fragment_admin.xml
Comment (delaroy) [inline app/src/main/res/layout/fragment_admin.xml:1]: use constraint layout

File: app/src/main/res/layout/fragment_create_admin.xml
Comment (delaroy) [inline app/src/main/res/layout/fragment_create_admin.xml:1]: same here

**Review by delaroy — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/tapshopping/domain/ShoppingRepositoryImpl.kt
Comment (whyhay01) [reply app/src/main/java/com/example/tapshopping/domain/ShoppingRepositoryImpl.kt:1]: This has been resolved

File: app/src/main/java/com/example/tapshopping/ui/viewModel/AdminViewModel.kt
Comment (whyhay01) [reply app/src/main/java/com/example/tapshopping/ui/viewModel/AdminViewModel.kt:1]: This has been resolved

File: app/src/main/java/com/example/tapshopping/data/model/DummyData.kt
Comment (whyhay01) [reply app/src/main/java/com/example/tapshopping/data/model/DummyData.kt:1]: Dummy is taken out

File: app/src/main/res/layout/fragment_admin.xml
Comment (whyhay01) [reply app/src/main/res/layout/fragment_admin.xml:1]: Resolved

File: app/src/main/res/layout/fragment_create_admin.xml
Comment (whyhay01) [reply app/src/main/res/layout/fragment_create_admin.xml:1]: Resolved

**Review by delaroy — APPROVED** [review]

**Pull request merged**